lightning/src/ln/channelmanager.rs

```diff
@@ -7248,12 +7311,14 @@ impl_writeable_tlv_based_enum!(PendingHTLCRouting,
 		(1, phantom_shared_secret, option),
 		(2, incoming_cltv_expiry, required),
 		(3, payment_metadata, option),
+		(5, custom_tlvs, option),
```

Comment:
> we should note in the release notes

Added a note to `pending_changelog` (seems like that's where this might go?) - noted it for 0.0.116 but ofc may end up being in 117